lumen/ai/actor.py

```diff
@@ -14,6 +15,9 @@
 
 class Actor(param.Parameterized):
 
+    interface = param.ClassSelector(class_=ChatInterface, doc="""
```

Comment:
Keeping tools separate from an interface was a deliberate decision so we keep blurring the line between a Tool and an Agent.